tests/python_package_test/test_plotting.py

```diff
@@ -68,6 +68,13 @@ def test_plot_importance(params, breast_cancer_split, train_data):
     assert ax2.patches[2].get_facecolor() == (0, .5, 0, 1.)  # g
     assert ax2.patches[3].get_facecolor() == (0, 0, 1., 1.)  # b
 
+    ax3 = lgb.plot_importance(gbm0, title='t @importance_type@', xlabel='x @importance_type@', ylabel='y @importance_type@')
+    assert isinstance(ax3, matplotlib.axes.Axes)
+    assert ax3.get_title() == 't @importance_type@'
```

Comment:
I'm confused by these tests. Shouldn't the template string `@importance_type@` have been replaced with the actual value of `importance_type`?

Reply:
Only in `xlabel` argument, not in `title` or any other.
https://github.com/microsoft/LightGBM/blob/6714a0f7fcfb2f5f0ea1adfa4c45b4ef0501098c/python-package/lightgbm/plotting.py#L62-L65

Reply:
ah I see, think I was confused by changes to the multiple different functions. Thanks for the explanation.